### PR TITLE
Fixes for 1.9 (Couldn't find mcmmo maven repo)

### PR DESCRIPTION
**Pull request by ChanceSD**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -63,38 +63,38 @@
 
 
     <dependencies>
-        <dependency>
+<!--         <dependency>
```
Comment (dualspiral) [inline pom.xml:66]: No replacement library or repository?

File: src/main/java/uk/co/drnaylor/mcmmopartyadmin/commands/FixPartiesCommand.java [hunk 2]
```diff
@@ -53,7 +53,7 @@ public boolean onCommand(CommandSender cs, Command cmnd, String string, String[]
 
         for (Party p : parties) {
             // Avoiding concurrent modifcation exceptions
-            List<String> pl = new ArrayList<String>(p.getMembers());
+			final List<String> pl = new ArrayList<String>(p.getMembers().values());
```
Comment (dualspiral) [inline src/main/java/uk/co/drnaylor/mcmmopartyadmin/commands/FixPartiesCommand.java:56]: Formatting - spaces.

File: src/main/java/uk/co/drnaylor/mcmmopartyadmin/commands/subcommands/ListPartiesSubCommand.java [hunk 3]
```diff
@@ -1,121 +1,125 @@
...
+import com.gmail.nossr50.api.PartyAPI;
+import com.gmail.nossr50.datatypes.party.Party;
+
 import uk.co.drnaylor.mcmmopartyadmin.PartyAdmin;
 import uk.co.drnaylor.mcmmopartyadmin.dualcommandexecutor.DualSubCommandInterface;
 import uk.co.drnaylor.mcmmopartyadmin.locales.L10n;
 
 public class ListPartiesSubCommand implements DualSubCommandInterface {
 
-    private List<String> permissions = new ArrayList<String>();
```
Comment (dualspiral) [inline src/main/java/uk/co/drnaylor/mcmmopartyadmin/commands/subcommands/ListPartiesSubCommand.java:18]: Again, your formatting has completely removed any chance I can see of seeing the change. Please make sure the diff is minimal, keep the formatting please.

File: src/main/java/uk/co/drnaylor/mcmmopartyadmin/dualcommandexecutor/DualCommandExecutor.java [hunk 4]
```diff
@@ -128,14 +127,13 @@ public final boolean onCommand(CommandSender sender, Command command, String lab
      * subcommand, if any.
      * @return Should normally be true.
      */
-    @Override
```
Comment (dualspiral) [inline src/main/java/uk/co/drnaylor/mcmmopartyadmin/dualcommandexecutor/DualCommandExecutor.java:131]: Don't remove this.

File: src/main/java/uk/co/drnaylor/mcmmopartyadmin/dualcommandexecutor/DualCommandExecutor.java [hunk 5]
```diff
@@ -98,7 +98,6 @@ public final void UnregisterSubCommand(String subcommand) throws SubCommandNotRe
...
```
Comment (dualspiral) [inline src/main/java/uk/co/drnaylor/mcmmopartyadmin/dualcommandexecutor/DualCommandExecutor.java:101]: Don't remove this.

**Comment by dualspiral** [comment]
While I appreciate your PR as I have little to no interest in maintaining it myself, please don't mess with the formatting if your intention is to submit it here. I'm still interested in what changes were made - and it's incredibly difficult to see what had needed to be changed.

Alternatively, you (or anyone else) are welcome to just keep your own fork updated - the code is licensed under the GPL - so you must keep it open. Should anyone wish to compile and submit it to plugin repositories, that's up to you, but I ask that you do not try to sell it - the Minecraft modding community should be open and I object to my code being sold when it was designed to be open in the first place.

**Comment by ChanceSD** [comment]
Uh, it's pretty easy to see what are the changes except in 1 or 2 classes, just adding ".values()". Also, what was removed had to be removed or has no effect.
So yea, i tried to not mess with formatting but i really don't care, just don't accept if you don't want it. Also, nobody talked about selling anything, you seem pretty selfish talking about that since you don't use the code or will use it anytime soon.
I got it working on my server and that's enough for me.

**Pull request closed without merging**